roles/bigboot/files/bigboot.sh

```diff
@@ -184,6 +185,22 @@ ensure_extendable_fs_type() {
     BOOT_FS_TYPE=$ret
 }
 
+get_nolocking_opts() {
+    local lvm_version
+    lvm_version="$(/usr/sbin/lvm version | grep 'LVM version:')"
+    status=$?
+    if [[ $status -ne 0 ]]; then
+        echo "Error getting LVM version '$lvm_version'"
+        exit $status
+    fi
+    # true when LVM version is older than 2.03
+    if echo -e "${lvm_version##*:}\n2.03" | sed 's/^ *//' | sort -V -C; then
```

Comment:
To avoid potential path issues, the script uses full path for executables (`sed` and `sort`)

Reply:
Good catch, thanks!